docs/reference/cluster/reroute.asciidoc

```diff
@@ -12,7 +12,7 @@ Changes the allocation of shards in a cluster.
 
 `POST /_cluster/reroute`
 
-
+[role="child_attributes"]
```

Comment:
This attribute should be included in the sections containing the collapsible sections.
```suggestion
```